flowtorch/parameters/coupling.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) Meta Platforms, Inc
+
+import warnings
+from typing import Callable, Optional, Sequence
+
+import torch
+import torch.nn as nn
+from flowtorch.nn.made import create_mask, MaskedLinear
+from flowtorch.parameters.base import Parameters
+
+
+class DenseCoupling(Parameters):
+    autoregressive = True
```

Comment:
I may be misremembering how this works, but doesn't a coupling layer use a dense NN rather than an autoregressive one?